src/main/webapp/script.js

```diff
@@ -241,3 +241,21 @@ function updateRestaurantInfo(restaurant, marker) {
   infowindow.setContent(infowindowContent);
   infowindow.open(map, marker);
 }
+
+document.getElementById("resize-nav-button").addEventListener("click", function() {
+  resizeNavButton();
+});
+
+
+function resizeNavButton(keepCollapsed=false) {
+  let button = document.getElementById("resize-nav-button");
+  let topNavBar = document.getElementById("pac-card");
+  if (keepCollapsed || !topNavBar.style.maxHeight) {
+    topNavBar.style.maxHeight = "105px";
```

Comment:
How did you come up with the value 105px ?

Reply:
It's the max height that both looked good and would still hide the filter segment without leaking any lines of it.

I increased the top margin of the filter segment (#85 in style.css), increasing the max height, to make it look better.

Reply:
okay it looks good on my side! i am not sure whether the 105px max height value would look similar for different devices / browsers.

Reply:
Looks okay from my testing so far, works with different zoom levels and in mobile